packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1976,29 +1975,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return context.realize();
     }
 
-    public setFlushMode(mode: FlushMode): void {
-        if (mode === this._flushMode) {
-            return;
-        }
-
-        this.mc.logger.sendTelemetryEvent({
-            eventName: "FlushMode Updated",
-            old: this._flushMode,
-            new: mode,
-        });
-
-        // Flush any pending batches if switching to immediate
-        if (mode === FlushMode.Immediate) {
-            this.flush();
-        }
-
-        this._flushMode = mode;
-
-        // Let the PendingStateManager know that FlushMode has been updated.
-        this.pendingStateManager.onFlushModeUpdated(mode);
-    }
-
-    public flush(): void {
+    /**
+     * Flush the pending ops manually.
+     * This method is expected to be called at the end of a batch.
+     * @param isImmediateBatch - is this "flush" being called at the end of a "FlushMode.Immedite" batch?
```

Comment:
```suggestion
     * @param isImmediateBatch - is this "flush" being called at the end of a "FlushMode.Immediate" batch?
```

Reply:
Do we need to expose this publicly? If we're the only ones to be calling `flush` with `isImmediateBatch === true` maybe flush with parameter can be private and the public flush will always be with false? Just trying to see if we can decrease the surface area of the API even more.

Reply:
I think it's fine for now since `flush` has been deprecated. I'll revisit this in [AB#1076](https://dev.azure.com/fluidframework/internal/_workitems/edit/1076)